kroxylicious-schema-validation/src/main/java/io/kroxylicious/proxy/filter/schema/config/TopicMatchingRecordValidationRule.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.filter.schema.config;
+
+import java.util.Objects;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Configuration that links a topic name pattern with some record validation rules.
+ */
+public class TopicMatchingRecordValidationRule extends RecordValidationRule {
+
+    private final Set<String> topicNames;
```

Comment:
Did you think of applying [kafka consumer conventions](topicNames) are let this be specified by a [pattern](https://kafka.apache.org/0100/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#subscribe(java.util.regex.Pattern,%20org.apache.kafka.clients.consumer.ConsumerRebalanceListener))?